oasis/common/io.py

```diff
@@ -91,16 +91,14 @@ def save_tstep_solution_h5(tstep, q_, u_, newfolder, tstepfiles, constrained_dom
         # project or store velocity to vector function space
         for comp, tstepfile in six.iteritems(tstepfiles):
             if comp == "u":
-                V = q_['u0'].function_space()
-                # First time around create vector function and assigners
-                if not hasattr(tstepfile, 'uv'):
-                    tstepfile.uv = AssignedVectorFunction(u_)
+                # Create vector function and assigners
+                uv = AssignedVectorFunction(u_)
```

Comment:
Only problem here is that now this vector function will be created new every time. That is not efficient, but then again, it's not a big deal. Can be fixed later on.